src/content/blog/astro-430.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Astro 4.3"
+description: "Astro 4.3 is out now! This release includes support for domain routing in i18n, better support for relative images in Markdown, a new `ComponentProps` type export, and more."
+publishDate: "February 2, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+coverImage: "/src/content/blog/_images/astro-430/post-header-4.3.webp"
+socialImage: "/src/content/blog/_images/astro-430/og-image-4.3.webp"
+lang: "en"
+---
+
+Astro 4.3 is now available! This release includes a new experimental i18n feature to try out, and improvements to working with your build output, component prop types, Markdown images, and more.
+
+Highlights include:
+
+- **[Add domain support for i18n](#experimental-add-domain-support-for-i18n)**
```

Comment:
Makes this bullet match the heading, which seemed to be what other posts do too:

```suggestion
- **[Experimental: Add domain support for i18n](#experimental-add-domain-support-for-i18n)**
```